common.js

```diff
@@ -40,11 +40,11 @@ function getPoints(userId, callback) {
 function getPendingPoints(userId, callback) {
     var outgoing = `SELECT SUM(u.points) AS totalPoints
     FROM 
-    (SELECT (SUM(Transactions.pointCost) * -1) AS points 
+    (SELECT (SUM(Transactions.buyerPoints) * -1) AS points 
     FROM Transactions 
     WHERE requestorId = ? AND statusId <> 8 
     UNION 
-    SELECT SUM(t.pointCost) AS points 
+    SELECT SUM(t.buyerPoints) AS points 
```

Comment:
I think this should be sellerPoints. It shouldn't make a huge difference in the results but since this select statement is pulling books where they are the seller we should use sellerPoints.